docs/documentation.md

```diff
@@ -111,22 +111,18 @@ var search = instantsearch({
 });
 {% endhighlight %}
   </div>
-  <div class="jsdoc js-toggle-jsdoc" style="display: none">
+  <div class="jsdoc" style="display: none">
 {% highlight javascript %}
 instantsearch(options);
 {% endhighlight %}
 
 {% include widget-jsdoc/instantsearch.md %}
   </div>
-  <div class="requirements js-toggle-requirements">
-Make sure you are using the **search-only API key** and that you have created
-the index.
-  </div>
 </div>
 
 </div>
 
-If you don't have any index yet, learn how to push your data with our [getting started guide](https://www.algolia.com/getstarted).
+If you don't have any indices yet, learn how to push your data with the [Algolia getting started](https://www.algolia.com/getstarted).

```

Comment:
Yes, `index` is the singular form and `indices` is its plural. (another plural would be `indexes` but this one is used in less technical context -> unfortunately our API uses `indexes` at some points).